notebooks/reinforcement_learning/solutions/contextual_bandits_with_tf_agents.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>Can we make the comment to say what is expected when we inspect the reward_spec(), observation_spec() and action_spec()?</p><p>Ex: #TODO Inspect the observation_spec to ensure that the shape=(NUMBER_OF_FEATURE_COLUMNS, )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>